Makefile

```diff
@@ -62,8 +63,10 @@ tidy:
 
 format:
 	@echo ">> formatting code"
-	@$(GO) fmt $(pkgs)
-	@cd cmd && $(GO) fmt $(cmd_pkgs)
+	@# Can't use $(pkgs) and $(cmd_pkgs) here as we need
+	@# to skip auto-generated files.
+	@find -name '*.go' | grep -Ev '^vendor/|.pb.go$$' | \
```

Comment:
Had to move jobs around a bit, but the coverage is the same.

Now golangci-lint is required on the client side; I think developers can figure it out how to install it.

Reply:
thanks for the cleanup! Now that `golangci-lint` is required on client side, can we perhaps change the lint target:

```
lint:
    @echo ">> running golangci-lint using configuration at .golangci.yml"
    @GOFLAGS="$(GO_FLAGS)" $(go_path)/bin/golangci-lint run
```

to run `golangci-lint` via `docker run` and mount cAdvisor source inside, so that way we won't require devs to install it (and the only dependency would be a docker install?).

Reply:
I don't know. I have golangci-lint installed locally and I guessed most of golang devs have it installed as it's pretty useful to run a bunch of linters at once. Installation is also trivial, it's one liner, and all platforms are supported (https://golangci-lint.run/usage/install/#local-installation).

Using docker image is also ok, I guess. Do you mind if I make it conditional (ie only use it if there is no binary found in $PATH)?

Reply:
Ah! Thanks for clarification, will implement.